be/src/formats/parquet/column_converter.cpp

```diff
@@ -744,9 +740,12 @@ Status Int64ToDateTimeConverter::convert(const ColumnPtr& src, Column* dst) {
         if (!src_null_data[i]) {
             int64_t seconds = src_data[i] / _second_mask;
             int64_t nanoseconds = (src_data[i] % _second_mask) * _scale_to_nano_factor;
-            TimestampValue ep;
-            ep.from_unixtime(seconds, nanoseconds / 1000, _ctz);
-            dst_data[i].set_timestamp(ep.timestamp());
+
+            std::chrono::system_clock::time_point tp = std::chrono::system_clock::from_time_t(seconds);
```

Comment:
I hope in application level,  we don't need to handle those timepont stuff.

do you think if you can extract two functions
- get_offset_by_seconds_since_epoch(secodns_since_epoch, cctz)
- get_offset_by_timestamp(Timestamp, cctz)